index.js

```diff
@@ -4,15 +4,139 @@
 
   /*istanbul ignore next*/
   if(module && typeof module.exports !== 'undefined'){
-    module.exports = f();
+    module.exports = f(require('sanctuary-type-classes'), require('sanctuary-type-identifiers'));
   }else{
-    global.concurrify = f();
+    global.concurrify = f(global.sanctuaryTypeClasses, global.sanctuaryTypeIdentifiers);
   }
 
-}(/*istanbul ignore next*/(global || window || this), function(){
+}(/*istanbul ignore next*/(global || window || this), function(Z, type){
 
   'use strict';
 
-  return function(){};
+  var $alt = 'fantasy-land/alt';
+  var $ap = 'fantasy-land/ap';
+  var $map = 'fantasy-land/map';
+  var $of = 'fantasy-land/of';
+  var $zero = 'fantasy-land/zero';
+  var $$type = '@@type';
+
+  var ordinal = ['first', 'second', 'third', 'fourth', 'fifth'];
+  var show = Z.toString;
+
+  function isFunction(f){
+    return typeof f === 'function';
+  }
+
+  function isBinary(f){
+    return f.length >= 2;
+  }
+
+  function isRepr(_){
+    return true;
+  }
```

Comment:
You, conditional `$equals` method (and `$concat`?) would be nice. I pinged @davidchambers with the question if it would be feasible to add something like `Z.isRepr`.